docs/android_sqlite/index.md

```diff
@@ -13,6 +13,7 @@ buildscript {
   }
 }
 
+apply plugin: 'kotlin'
```

Comment:
```suggestion
apply plugin: 'org.jetbrains.kotlin.android'
```

Reply:
using `org.jetbrains.kotlin.android` throws this error for me

> Failed to apply plugin [id 'org.jetbrains.kotlin.android']
   > Extension with name 'android' does not exist. Currently registered extension names: [ext, kotlin, kotlinTestRegistry]

Reply:
The Android plugin would also need applied. Either `com.android.library` or `com.android.application`.